var/spack/repos/builtin/packages/elfutils/package.py

```diff
@@ -54,6 +54,8 @@ class Elfutils(AutotoolsPackage, SourcewarePackage):
     variant('debuginfod', default=False,
             description='Enable libdebuginfod support.')
 
+    patch('elfutils-0.185-static-inline.patch', when='@0.185')
```

Comment:
Can you add a brief comment over the patch to explain what it does and where it was taken from? Bonus if you can remove the patch itself in Spack and retrieve it directly from:
- https://794601.bugs.gentoo.org/attachment.cgi?id=714030

Reply:
Done!